hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -507,12 +509,12 @@ private boolean checkKeyReclaimable(
       String dbKey;
       // Handle case when the deleted snapshot is the first snapshot.
       if (previousKeyTable == null) {
-        return false;
+        return true;
```

Comment:
I think we should call the method `isKeyReclaimable`/`areKeysReclaimable`

Reply:
Makes sense, will update the patch.